configs/nextflow.config

```diff
@@ -219,6 +219,36 @@ process {
     withLabel:time_queue_from_normal {
         time = { check_max( escalate_queue_time( 'normal', task ), 'time' ) }
     }
+
+    // Config for processes from https://gitlab.internal.sanger.ac.uk/sanger-pathogens/pipelines/assorted-sub-workflows
+    // for IRODS_EXTRACTOR subworkflow
+
+    withName:JSON_PREP {
+        executor = "local"
+    }
+
+    withName:BATON {
+        executor = "local"
+    }
+
+    withName:JSON_PARSE {
+        executor = "local"
+    }
+
+    withName:RETRIEVE_CRAM {
+        executor = "lsf"
+        clusterOptions = "-R 'rusage[seqirods=100]'"
+    }
+}
+
+subworkflows {
+
+    // Config for subworkflows from https://gitlab.internal.sanger.ac.uk/sanger-pathogens/pipelines/assorted-sub-workflows
+    // for IRODS_EXTRACTOR subworkflow
+    
+    withName:IRODS_QUERY {
+        maxForks = 1
```

Comment:
I would be inclined to put `maxForks = 1` at the process level for the `BATON` process as that's the only one doing the iRODS querying in the workflow.

Reply:
agreed; done